### PR TITLE
21 wof header

### LINKED ISSUE #21: Create WofHeader component
Header for our App, based on design on Figma.
Should contain Logo placeholder, placeholder link to Sign In, placholder for user avatar/user icon image
Should be placed in App.vue as a component.

### PR DESCRIPTION
**Pull request by karolkozlo**
How to test: Just look at main page. Change 'user' data value to see different links
closes #21

### CODE REVIEW

File: frontend/src/components/WofHeader.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      }
+
+      .header__link-text {
+        font-size: 1.2rem;
+        margin-right: 0.8em;
+      }
+    }
+  }
+}
+</style>
```
Comment (kpagacz) [inline frontend/src/components/WofHeader.vue:83]: What's up with the missing endlines?

File: frontend/src/components/WofHeader.vue [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      <img class="header__logo" src="../assets/logo.png" />
+      <router-link to="/" class="header__main-link">Wiki On Fire</router-link>
+    </div>
+    <div class="header__user-nav">
+      <router-link to="/login" class="header__nav-link" v-if="!user.length">
+        <span class="header__link-text">Sign In</span>
+        <wof-icon icon="wof-user" :size="2"></wof-icon>
+      </router-link>
+      <router-link to="/user/username" class="header__nav-link" v-if="user.length">
+        <span class="header__link-text">Username</span>
```
Comment (kpagacz) [inline frontend/src/components/WofHeader.vue:13]: Should it this be:
```suggestion
        <span class="header__link-text">{{ this.user }}</span>
```
?

**Review by kpagacz — APPROVED** [review]
Other than the small comment, it looks good.

File: frontend/src/components/WofHeader.vue (hunk 2 above)
Comment (karolkozlo) [reply frontend/src/components/WofHeader.vue:13]: We will add this after user logging in

File: frontend/src/components/WofHeader.vue (hunk 1 above)
Comment (karolkozlo) [reply frontend/src/components/WofHeader.vue:83]: I'm not used to it

**Pull request merged**